bin/l2l-fun-sa.py

```diff
@@ -58,42 +58,12 @@ def main():
 
     # Set JUBE params
     traj.f_add_parameter_group("JUBE_params", "Contains JUBE parameters")
+    # Execution command
+    traj.f_add_parameter_to_group("JUBE_params", "exec", "python3 " +
```

Comment:
I am wondering if `python3` is still needed as a command. I remember the call `python3` call made a few problems for me because it was not using the python from my environment but the system wide python.

Reply:
Yes indeed it would be a good idea to leave the examples with only python. I need this way of executing the files because of my own configuration but we should try to avoid confusions.